test/integration/targets/vmware_host_tcpip_stacks/tasks/main.yml

```diff
@@ -0,0 +1,77 @@
+# Test code for the vmware_host_tcpip_stacks module.
+# Copyright: (c) 2018, Christian Kotte <christian.kotte@gmx.de>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+- name: store the vcenter container ip
+  set_fact:
+    vcsim: "{{ lookup('env', 'vcenter_host') }}"
+
+- debug: var=vcsim
+
+- name: Wait for Flask controller to come up online
+  wait_for:
+    host: "{{ vcsim }}"
+    port: 5000
+    state: started
+
+- name: kill vcsim
+  uri:
+    url: http://{{ vcsim }}:5000/killall
+
+- name: start vcsim
+  uri:
+    url: http://{{ vcsim }}:5000/spawn?cluster=2
+  register: vcsim_instance
+
+- debug:
+    var: vcsim_instance
+
+- name: Wait for vcsim server to come up online
+  wait_for:
+    host: "{{ vcsim }}"
+    port: 443
+    state: started
+
+- name: get a list of Cluster from vcsim
+  uri:
+    url: http://{{ vcsim }}:5000/govc_find?filter=CCR
+  register: clusters
+
+- name: get a cluster
+  set_fact:
+    ccr1: "{{ clusters.json[0] | basename }}"
+
+- name: get a list of hosts from vcsim
+  uri:
+    url: http://{{ vcsim }}:5000/govc_find?filter=H
+  register: hosts
+
+- name: get a host
+  set_fact:
+    host1: "{{ hosts.json[0] | basename }}"
+
+- debug: var=ccr1
+- debug: var=host1
+
+- name: Configure TCP/IP stacks
+  vmware_host_tcpip_stacks:
```

Comment:
Could you please again run this task so as to check idempotence of this module ? Thanks

Reply:
Any specific reason? I changed unicode to six to get rid of the Flake8 warning...

Reply:
Test will check idempotency of module.

Reply:
It is good that module is idempotent in your environment. I just want to make sure that it stay idempotent in testing and other users' environment. Imagine a scenario where somebody adds a patch to this and breaks the idempotency in future, so this test case will make sure that it retains that.

Reply:
Yes, I just don't understand what I should change. TCP/IP stacks can't be configured with vcsim. 

This task configures everything except the default gateway. Do you want me to add the default gateway to one of the stacks, except the default stack!?

Reply:
Ok Let it be !

Reply:
So update govmomi/simulator to remember what it is told, and then run the task a second time in the integration test, and it will test idempotency, and more importantly a fair number of branches that would otherwise be untested.  You want to assert that changed is False after running the task a second time with the same parameters.